brayovsky/pivotaly
# Show story details in backlog 171254007

#### What does this PR do? / Fixes issue
Adds story details to backlog stories

#### How should this be manually tested?
Open current and backlog view.
Each item should be expandable to allow viewing of the story's details

#### Any background context you want to provide?
n/a
#### Screenshots (if appropriate)
![Screenshot from 2020-04-09 22-48-19](https://user-images.githubusercontent.com/19430730/78934879-62e38000-7ab4-11ea-90a6-bedbef0eeb2b.png)

#### Questions:
n/a
